comp/languagedetection/client/util.go

```diff
@@ -0,0 +1,120 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package client holds the client to send data to the Cluster-Agent
+package client
+
+import (
+	pbgo "github.com/DataDog/datadog-agent/pkg/proto/pbgo/process"
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+)
+
+type languageSet map[string]struct{}
```

Comment:
I suspect we already have this type in @adel121 PR.

Reply:
I fixed it !